cmd/autotestgridgenerator/main.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/experiment/autobumper/bumper"
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/flagutil"
+	"k8s.io/test-infra/prow/labels"
+)
+
+const (
+	githubOrg    = "kubernetes"
+	githubRepo   = "test-infra"
+	githubLogin  = "openshift-bot"
+	githubTeam   = "openshift/openshift-team-developer-productivity-test-platform"
+	matchTitle   = "Automate testgrid generator"
+	remoteBranch = "auto-testgrid-generator"
+)
+
+type options struct {
+	selfApprove bool
+
+	githubLogin       string
+	gitName           string
+	gitEmail          string
```

Comment:
The embedded `flagutil.GithubOptions` already has these.

Reply:
I cant seem to find them, I can see a "Host" field in there. Is that what you meant?

Reply:
Sorry, I misread. My mistake.